java/test/jmri/jmrit/logix/LearnWarrantTest.java

```diff
@@ -33,11 +34,12 @@
 public class LearnWarrantTest {
 
     @Rule
-    public RetryRule retryRule = new RetryRule(3);  // allow 3 retries
+    public RetryRule retryRule = new RetryRule(2);  // allow retry
 
     private OBlockManager _OBlockMgr;
 
     @Test
+    @Ignore("Sometimes causes InvocationTargetException at OBlock line 932")
```

Comment:
Looking at one of the logs from this PR, I see:

java.lang.NullPointerException
743	at jmri.jmrit.logix.OBlock.lambda$goingInactive$1(OBlock.java:933)
744	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
745	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
746	at java.awt.EventQueue.access$500(EventQueue.java:97)
747	at java.awt.EventQueue$3.run(EventQueue.java:709)
748	at java.awt.EventQueue$3.run(EventQueue.java:703)
749	at java.security.AccessController.doPrivileged(Native Method)
750	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
751	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
752	at org.netbeans.jemmy.QueueTool$JemmyQueue.dispatchEvent(QueueTool.java:656)
753	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
754	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
755	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
756	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
757	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
758	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
759ERROR -    Came from call to runOnGUI: [main] jmri.util.ThreadingUtil.runOnGUI()
760java.lang.reflect.InvocationTargetException
761	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1349)
762	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1324)
763	at javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1353)
764	at jmri.util.ThreadingUtil.runOnGUI(ThreadingUtil.java:116)
765	at jmri.util.ThreadingUtil.runOnLayout(ThreadingUtil.java:40)
766	at jmri.jmrit.logix.OBlock.goingInactive(OBlock.java:932)
767	at jmri.Block.handleSensorChange(Block.java:630)

So there is a null pointer exception occurring before the InvocationTargetException.  

Looking at the code in question:

https://github.com/petecressman/JMRI/blob/4d78b918ce82633c1eed74e125e0f3ba33ce644d/java/src/jmri/jmrit/logix/OBlock.java#L931-L935

The call to runOnGui throws the exception because it caught an error in the code it was running, which is the goingInactive call.  I haven’t debugged it further than that, but that is where you should look to fix the problem.